jekyll/_cci2/caching.md

```diff
@@ -370,7 +370,7 @@ Since caches are immutable, this strategy allows you to regenerate all of your c
 
 The stability of partial dependency caching is dependent on your dependency manager. Below is a list of common dependency managers, recommended partial caching strategies, and associated justifications.
 
-#### Bundler (Ruby)
+#### Bundler (ruby)
```

Comment:
Should languages be capitalised? 🤔

Reply:
Probably - can you make suggestion change and I'll commit it? I don't think I can merge my own suggestions on a PR.